decidim-core/db/migrate/20200323094443_add_id_to_content_blocks_scope.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddIdToContentBlocksScope < ActiveRecord::Migration[5.2]
+  def change
+    add_column :decidim_content_blocks, :scope_id, :integer
```

Comment:
`scope_id` is very confusing because Decidim has the concept of Sope as a model, but I can't think of an alternative: `parent_id`, `related_id`, ...

Reply:
Yeah, me neither... 🤷

Reply:
can we use `container_id` so we get rid of the confusion?

Reply:
`container_id` makes me think that the container block is contained inside another resource... What about `resource_id`?